carma_wm/src/Geometry.cpp

```diff
@@ -25,6 +25,8 @@ namespace carma_wm
 namespace geometry
 {
 
+constexpr double SPATIAL_EPSILON_M = 0.01;
```

Comment:
Lets make this 0.05

I saw one extra set of duplicates while evaluating in excel by changing the epsilon to 0.05 but higher values didn't result in more